content/trial_getting_started/examples/kubernetes/_index.md

```diff
@@ -0,0 +1,99 @@
+
+### Service Composition
+
+The configuration indirection helps with mapping simple identifiers (like 'dev-small') to complex configuration details (like image names/id, flavors, etc..).  This is not sufficient to completely abstract away the different Kubernetes environments required.  To do this requires [service composition](../../../working_with/service_composition).  Service composition allows blueprint nodes to represent entire blueprints themselves, effectively nesting blueprints and enabling a building block approach.  The main requirement to be able to use components in the way required by this example is that components of a similar kind (for example, blueprints that represent different kinds of database), all have a consistent interface.  In Cloudify DSL, this interface is provided by the `capabilitites` section in the blueprint.
+
+Looking at the [multi-Kubernetes cluster example](https://github.com/cloudify-community/eaas-example), consider the object storage options `minio` and `S3`.  Both of these are represented by a blueprint that exposes a single `capability`: `bucket_url`.  Because of this standard "interface", the blueprints can be substituted for each other at deploy time.  You will find the same pattern for other elements: minikube/EKS (`endpoint`), psql/RDS (`host`, `master_username`, `master_password`), and so on.
+
+![Kubernetes Multi-Cluster Use Case]( /images/k8s_multicluster/multik8s.png )
+
+## Running the Example Implementation
+
+### Prerequisites
+This example expects the following prerequisites:
+
+* A {{< param cfy_manager_name >}} setup ready. This can be either a [{{< param mgr_hosted_title >}}]({{< param mgr_hosted_link >}}), a [{{< param mgr_premium_title >}}]({{< param mgr_premium_link >}}), or a [{{< param mgr_community_title >}}]({{< param mgr_community_link >}}).
+* The following plugins must be uploaded to the {{< param cfy_manager_name >}}
+  * AWS Plugin (version 2.5.6+)
+  * Kubernetes Plugin (version 2.9.3+) 
+  * Terraform Plugin (version 0.15.0+)
+  * Fabric Plugin (version 2.0.7+)
+* The following secrets must be defined on the {{< param cfy_manager_name >}}
+  * `aws_access_key_id` - The AWS access key
+  * `aws_secret_access_key` - The AWS secret key
+  * `aws_keypair` - The name of an AWS keypair to associate with virtual machines.  The key must be created on AWS prior to running the example.
+  * `private_key_content` - The SSH private key contents from the keypair
+  
```

Comment:
Let's update this section to be consistent with how other examples are leveraging key pairs:
https://github.com/cloudify-community/blueprint-examples/blob/master/kubernetes/aws-eks/blueprint.yaml#L71. This way the user does not need to be responsible for creating and importing a key pair.